website/docs/release_notes.md

```diff
@@ -7,6 +7,75 @@ import ReactPlayer from 'react-player'
 
 # Release notes
 
+## [v1.0.0](https://github.com/VirtusLab/scala-cli/releases/tag/v1.0.0)
+
+## The official `scala` runner release
+
+Scala CLI has reached the highly anticipated `1.0.0` milestone!
+Having addressed all the [SIP-46](https://github.com/scala/improvement-proposals/pull/46) requirements,
+this version is going to become the official `scala` runner, replacing the old `scala` command.
+
+For a deeper understanding of Scala CLI as the new `scala` runner and to explore its benefits and features,
+we encourage you to check out our [blogpost](https://virtuslab.com/blog/scala-cli-the-new-scala-runner/).
+
+Also be sure to get familiar with all the differences introduced by this change in our [migration guide](docs/guides/old-runner-migration.md).
+
+## What's Changed
+
+### Toolkit-test
+
+By incorporating the [Scala Toolkit](https://github.com/scala/toolkit) into your project, you gain the advantage of two additional
+dependencies seamlessly integrated into your classpath:
+- `org.scala-lang:toolkit:<version>` is added to the main scope, allowing its utilization throughout your project.
+- `org.scala-lang:toolkit-test:<version>` is included in the test scope, making it available exclusively for testing purposes.
+
+Scala CLI now supports the following features for the toolkit:
+* including e.g. `//> using toolkit latest` in any main scope file will automatically add the `toolkit` dependency to the main scope and the `toolkit-test` dependency to the test scope
+* if you place e.g. `//> using toolkit latest` within a test scope file, both `toolkit` and `toolkit-test` will be limited to the test scope only
+* inserting e.g. `//> using test.toolkit latest` anywhere in the project will add both `toolkit` and `toolkit-test` to the test scope only
+
+This convention is encouraged for other toolkit-like libraries as well.
+
+Added by [@Gedochao](https://github.com/Gedochao) in [#2127](https://github.com/VirtusLab/scala-cli/pull/2127) and [#2137](https://github.com/VirtusLab/scala-cli/pull/2137)
+
+### Forcing script wrapper type
+
+Scala CLI now supports the `//> using objectWrapper` directive, together with `--object-wrapper` option,
+which allows to force wrapping script code in an object body instead of a class.
```

Comment:
```suggestion
Scala CLI now supports the `//> using objectWrapper` directive, along with the corresponding `--object-wrapper` option,
which allows to force wrapping script code in an object body instead of a class.
```